### PR TITLE
Add ConfigUpdater

### PR DESCRIPTION
**Pull request by maresb**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [X] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by maresb** [comment]
@ocefpaf, this used to be a component of PyScaffold, and thus @FlorianWilhelm suggested that I contact you. Could I add you here as a recipe maintainer?

**Comment by maresb** [comment]
My first feedstock is ready for review, @conda-forge/help-python!

File: recipes/configupdater/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+test:
+  imports:
+    - configupdater
+  requires:
+    - pytest
+    - pytest-cov
+  source_files:
+    - tests
+  commands:
+    - pytest
```
Comment (chrisburr) [inline recipes/configupdater/meta.yaml:33]:
```suggestion
    - pytest
    - pip check
```

File: recipes/configupdater/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+
+test:
+  imports:
+    - configupdater
+  requires:
```
Comment (chrisburr) [inline recipes/configupdater/meta.yaml:27]: I forgot you'll also need this 😅 
```suggestion
  requires:
    - pip
```

**Comment by maresb** [comment]
Thanks @chrisburr, all checks passed. Ready?

**Comment by maresb** [comment]
@chrissburr, it didn't work. https://github.com/conda-forge/configupdater-feedstock/issues/1

**Comment by chrisburr** [comment]
Give it a few hours, it typically takes a few attempts for all of the CI providers to register correctly.

**Pull request merged**